_includes/frontpage/docs.html

```diff
@@ -1,39 +1,59 @@
 <section id="docs">
   <div class="container">
-    <h2>Documentation
-        <img class="shield" alt="GitHub package.json version"
+    <h2>Getting Started
+    <a href="https://github.com/MathJax/MathJax" class="shield">
+      <img class="shield" alt="MathJax version on GitHub"
            src="https://img.shields.io/github/package-json/v/mathjax/MathJax">
-       <img class="shield" alt="GitHub package.json version (branch)"
+      </img>
+    </a>
+    <a href="https://github.com/MathJax/MathJax/tree/legacy-v2" class="shield">
+      <img class="shield" alt="MathJax version 2 legacy version on GitHub"
            src="https://img.shields.io/github/package-json/v/mathjax/MathJax/legacy-v2">
+      </img>
+    </a>
+    <a href="https://www.npmjs.com/package/mathjax" class="shield">
+      <img class="shield" alt="MathJax npm package"
+           src="https://img.shields.io/npm/v/mathjax">
+      </img>
+    <a href="https://www.npmjs.com/package/mathjax-full" class="shield">
+      <img class="shield" alt="MathJax sources npm package"
+           src="https://img.shields.io/npm/v/mathjax-full?label=npm@sources">
```

Comment:
I'm wondering if `npm@full` is easier to tie to the `mathjax-full` package than `npm@source`?

Reply:
I've changed it to `npm@full`. But what we really want is something to indicate that this is for developers, whereas `npm` is for the casual user. Not sure how to write that.